packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -149,7 +150,7 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry {
    * @param attestationRequestFeeToken The address of the token with which the attestation fee will
    * be paid.
    */
-  function request(
+  function nonReentrant request(
```

Comment:
Modifier should go after `external`. I would have expected this to be picked up by the compiler and/or linter?

Reply:
I forgot to run the linter. This is fixed in a newer commit.